roles/lib_openshift/library/oc_adm_router.py

```diff
@@ -2937,6 +2945,7 @@ def main():
         mutually_exclusive=[["router_type", "images"],
                             ["key_file", "default_cert"],
                             ["cert_file", "default_cert"],
+                            ["cacert_file", "default_cert"],
                            ],
```

Comment:
It might be good to add required_together:
```
        required_together=[
            ['cacert_file', 'cert_file', 'key_file'],
        ],
```

It also looks like the cert arguments aren't covered in the docs.